aws-datasync-storagesystem/build.gradle

```diff
@@ -0,0 +1,95 @@
+buildscript {
+    dependencies {
+        classpath brazilGradle.tool('BrazilGradleQualityDefaults')
```

Comment:
I think you gotta get rid of all of the Brazil stuff from this file... (or remove the file entirely? idk)

Reply:
It looks like it needs to be removed entirely, could have sworn that it was meant to be included.